Detections/ZoomLogs/JoiningMeetingFromAnotherTimeZone.yaml

```diff
@@ -0,0 +1,40 @@
+id: 58fc0170-0877-4ea8-a9ff-d805e361cfae
+name: User joining Zoom meeting from suspicious timezone
+description: |
+  'The alert shows users that join a Zoom meeting from a time zone other than the one the meeting was created in.
+  You can also whitelist known good time zones in the tz_whitelist value using the tz database name format https://en.wikipedia.org/wiki/List_of_tz_database_time_zones'
+severity: Low
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes:
+      - ZoomLogs
+queryFrequency: 6h
+queryPeriod: 6h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Initial Access
+relevantTechniques:
+  - T1078
+query: |
+
+  let schedule_lookback = 30d; 
+  let join_lookback = 6h; 
```

Comment:
1d recommended here as it should be a daily alert, customers can adjust as they see fit.